.ansible-lint-ignore

```diff
@@ -0,0 +1,83 @@
+test/gce/scenarios/windows/tasks/create_windows_instance.yml fqcn[action-core]
+test/gce/scenarios/windows/tasks/create_windows_instance.yml yaml[line-length]
+test/gce/scenarios/windows/verify.yml fqcn[action-core]
+
+test/roles/azureplugin/meta/main.yml meta-incorrect
+test/roles/azureplugin/meta/main.yml meta-no-info
+test/roles/azureplugin/meta/main.yml schema[meta]
+test/roles/azureplugin/molecule/default/create.yml yaml[octal-values]
+test/roles/azureplugin/molecule/default/destroy.yml yaml[octal-values]
+test/roles/azureplugin/molecule/default/verify.yml yaml[indentation]
+test/roles/azureplugin/tests/test.yml name[play]
+
+test/roles/dockerplugin/meta/main.yml meta-incorrect
+test/roles/dockerplugin/meta/main.yml meta-no-info
+test/roles/dockerplugin/meta/main.yml schema[meta]
+test/roles/dockerplugin/molecule/default/converge.yml fqcn[action-core]
+test/roles/dockerplugin/molecule/default/verify.yml yaml[indentation]
+test/roles/dockerplugin/tests/test.yml name[play]
+
+test/roles/ec2plugin/meta/main.yml meta-incorrect
+test/roles/ec2plugin/meta/main.yml meta-no-info
+test/roles/ec2plugin/meta/main.yml schema[meta]
+test/roles/ec2plugin/molecule/default/converge.yml fqcn[action-core]
+test/roles/ec2plugin/molecule/default/create.yml fqcn[action-core]
+test/roles/ec2plugin/molecule/default/create.yml fqcn[action]
+test/roles/ec2plugin/molecule/default/create.yml fqcn[keyword]
+test/roles/ec2plugin/molecule/default/create.yml key-order[task]
+test/roles/ec2plugin/molecule/default/create.yml name[missing]
+test/roles/ec2plugin/molecule/default/create.yml no-handler
+test/roles/ec2plugin/molecule/default/create.yml risky-file-permissions
+test/roles/ec2plugin/molecule/default/destroy.yml fqcn[action-core]
+test/roles/ec2plugin/molecule/default/destroy.yml fqcn[action]
+test/roles/ec2plugin/molecule/default/destroy.yml fqcn[keyword]
+test/roles/ec2plugin/molecule/default/destroy.yml risky-file-permissions
+test/roles/ec2plugin/molecule/default/prepare.yml fqcn[action-core]
+test/roles/ec2plugin/molecule/default/verify.yml yaml[indentation]
+test/roles/ec2plugin/tests/test.yml name[play]
+
+test/roles/gceplugin/meta/main.yml meta-incorrect
+test/roles/gceplugin/meta/main.yml meta-no-info
+test/roles/gceplugin/meta/main.yml schema[meta]
+test/roles/gceplugin/molecule/default/verify.yml yaml[indentation]
+test/roles/gceplugin/tests/test.yml name[play]
+
+test/roles/podmanplugin/meta/main.yml meta-incorrect
+test/roles/podmanplugin/meta/main.yml meta-no-info
+test/roles/podmanplugin/meta/main.yml schema[meta]
+test/roles/podmanplugin/molecule/default/verify.yml yaml[indentation]
+test/roles/podmanplugin/tests/test.yml name[play]
+
+test/roles/vagrantplugin/meta/main.yml meta-incorrect
+test/roles/vagrantplugin/meta/main.yml meta-no-info
+test/roles/vagrantplugin/meta/main.yml schema[meta]
+test/roles/vagrantplugin/molecule/default/verify.yml yaml[indentation]
+test/roles/vagrantplugin/tests/test.yml name[play]
```

Comment:
Here is a long list. It's fun for us to fix the lint issue in another PR and finally, remove the related lines in this files in  new PR.

Reply:
I think in this PR we should leave this list of explicitely ignored errors then solve them in another PR

 FQDN and line length, i will do them in this PR, they are easy to solve

Reply:
I guess that meta errors are going to stay, unless the test script modifies the generated meta.yml file. Would have been easier if ansible-lint could support wildcards...